Add caveat about W3C on RDC delay (#949)
saucelabs/sauce-docs: docs/mobile-apps/automated-testing/appium/real-devices.md

## Using the W3C WebDriver Specification

:::caution W3C for RDC Delayed

> In order to support customers for whom the enablement of dual support for JWP and W3c will cause conflicts in active projects, we have delayed the release of W3C for real device testing on Appium until January 2022.

Author: ```suggestion
In order to support customers for whom the enablement of dual support for JWP and W3C will cause conflicts in active projects, we have delayed the release of W3C for real device testing on Appium until January 2022.
```

> In order to support customers for whom the enablement of dual support for JWP and W3C will cause conflicts in active projects, we have delayed the release of W3C for real device testing on Appium until January 2022.

Author: ```suggestion
In order to support customers who have active projects with conflicting configuration protocols that will be affected by the enablement of dual support for JWP and W3C, we have delayed the release of W3C for real device testing on Appium until January 2022.
```

:::

> 

Reviewer: Can we add to the sentence the following: "due to incompatible tests".
" In order to support customers for whom the enablement of dual support for JWP and W3C will cause conflicts in active projects **due to their incompatible test configurations**, we have delayed the release of W3C for real device testing on Appium until January 2022.

Otherwise the reason for the delay sounds to me as if Sauce is doing something wrong and will break their tests. But I am not a native speaker so this could be just me.

Author: That makes perfect sense. Thanks for the review!

Author: Is this better, @linafarid ? I didn't want it to sound too accusatory. :)

Reviewer: yup! sounds great!
